dashboard/src/main/resources/templates/macros.ftl

```diff
@@ -1,62 +1,54 @@
+<#function pluralize number singlarNoun pluralNoun>
+  <#local plural = number gt 1 />
+  <#return plural?string(number + " " + pluralNoun, number + " " + singlarNoun)>
```

Comment:
Does this work? `<#return number + " " + plural?string(pluralNoun, singularNoun)>`